samples/grids/grid/cell-editing-sample/src/index.tsx

```diff
@@ -38,6 +38,7 @@ export default class Sample extends React.Component<any, any> {
 
             <div className="container fill">
                 <IgrGrid
+                    ref={this.grid1Ref}
```

Comment:
That seems to be a generated sample though. If the ref is needed and has been removed by generation changes for 19 it might be an issue with the changes in the code generation.